app/uk/gov/hmrc/hmrcemailrenderer/templates/mandate/mandateAgentActivates.scala.html

```diff
@@ -0,0 +1,8 @@
+@(params: Map[String, Any])
+
+@uk.gov.hmrc.hmrcemailrenderer.templates.helpers.html.template_main(params, params("service").toString) {
+
```

Comment:
In order to avoid spam, it's helpful if there's a salutation in the email, for example "Dear Mr. John Smith"
I don't know whether your service has this information available: if possible, please use it.